benchmarks/benchmark_utils.py

```diff
@@ -0,0 +1,297 @@
+import sys

```

Comment:
Many of the utilities are used only in the context of benchmarking (they wrap our existing utilities), and some of them have very similar functionalities. Do we want to combine in a way to enhance the readability?

Reply:
I was thinking the same, but I wasn't sure to be honest how to extract the utilities to not create a black box for the user. I think we need to find a middle ground between adding too much content and having only few high level functions (load, train, evaluate)

Reply:
we can review this in a future issue